versioned_docs/version-3/community/localization.md

```diff
@@ -72,93 +70,137 @@ file.
+1. Add the language to the `i18n` key of the `docusaurus.config.js` file so that it appears in the dropdown in the navbar:
+    ```yaml
+    i18n: {
+      defaultLocale: 'en',
+      # add new language to this list of locales
+      locales: ['en', 'de', 'es', 'fr', 'gr', 'ja', 'ko', 'pt', 'ru', 'uk', 'zh'],
+      localeConfigs: {
+        en: {
+          htmlLang: 'en-us',
+          label: 'English',
+        },
+        de: {
+          label: 'Deutsch',
+        },
+      # new_lang {
+      #   label: 'Navbar label',
+      # }
+      },
+    },
+    ```
+1. (Optional) Translate docs or blog content. See _Translating_ below.
+1. Test your changes by starting the localized site in dev mode, specifying the locale:
+
+   ```
+   yarn start --locale fr
+   ```
+   :::note
+   Each locale is a distinct standalone single-page application. You can only preview one locale at a time. It is not possible to preview all locales at the same time.
+   :::
 
-The content in the new file can then be translated into the other language.
+### Translating
 
-Do not add an untranslated copy of an English file to `content/[LANG]/`.
+Before you translate docs content, review the following best practices and guidelines:
+* Translation tools can help with the process. This includes machine
+generated translations. Machine generated translations should be edited or
+otherwise reviewing for grammar and meaning by a native language speaker before
+publishing.
+* Do not add an untranslated copy of an English file to `i18n/[LANG]/plugin-content-docs` or `i18n/[LANG]/plugin-content-blog`.
 Once a language exists on the site, any untranslated pages will redirect to
 English automatically. Translation takes time, and you always want to be
 translating the most current version of the docs, not an outdated fork.
-
-Make sure you remove any `aliases` lines from the header section. A line like
+* Make sure you remove any `aliases` lines from the header section. A line like
 `aliases: ["/docs/using_helm/"]` does not belong in the translations. Those
 are redirections for old links which don't exist for new pages.
-
-Note, translation tools can help with the process. This includes machine
-generated translations. Machine generated translations should be edited or
-otherwise reviewing for grammar and meaning by a native language speaker before
-publishing.
-
+* Add anchor IDs to any headings using the format `## Example Heading {#example-anchor-id}`. The anchor IDs must be English language and match the anchor IDs of the corresponding heading in the English doc page. For example, `## 后端存储 {#storage-backends}` matches `## Storage backends {#storage-backends}`. This ensures that any anchor links to the headings still work in the translated version of the page. For more information, see [Anchor links to headings](https://github.com/helm/helm-www/blob/hip-0024-helm4-site-upgrade/ARCHITECTURAL_DECISIONS.md#anchor-links-to-headings) in `ARCHITECTURAL_DECISIONS.md`.
```

Comment:
^ added this info about anchor links

note that this link is broken right now but will work once it's on main